Makefile

```diff
@@ -1,13 +1,20 @@
+export GOGO_PROTOBUF := ${GOPATH}/pkg/mod/$(shell go mod graph | grep github.com/gogo/protobuf | sed -e 's/github.com\/pingcap\/tipb //g')
```

Comment:
Is it possible to use `go list -f "{{.Dir}}" -m github.com/gogo/protobuf`?

Reply:
Wow, I think it works.